browser/src/core/Socket.js

```diff
@@ -1533,17 +1533,33 @@ app.definitions.Socket = L.Class.extend({
 		// Let onclose (_onSocketClose) report errors.
 	},
 
-	_onSocketClose: function () {
+	_onSocketClose: function (event) {
 		window.app.console.debug('_onSocketClose:');
 		if (!this._map._docLoadedOnce && this.ReconnectCount === 0) {
+			var errorMsg, errorType = '';
+			var reason = event.reason;
+			if (reason && reason.startsWith('error:')) {
+				var command = this.parseServerCmd(reason);
+				if (command.errorCmd === 'internal' && command.errorKind === 'unauthorized') {
+					errorType = 'websocketunauthorized';
+					errorMsg = errorMessages.unauthorized;
+				} else if (command.errorCmd === 'storage' && command.errorKind === 'loadfailed') {
+					errorType = 'websocketloadfailed';
+					errorMsg = errorMessages.storage.loadfailed;
+				}
```

Comment:
else ?
we need something generic here I expect. Because it will pass nothing to the post message

Reply:
but I can't find anything other error being sent from the server, but we should handle here

Reply:
yes but it might be changed in the future